### ISSUE TITLE
Xbox One controller left stick not responsive

### ISSUE THREAD

**Issue by OpaqueEcho** [KozGit/DOOM-3-BFG-VR#251]
I looked through previous posts and didn't find one that matched my issue; apologies in advance if I overlooked something!

I'm running the game otherwise without issue using Trinus PSVR and the PSVR HMD. Visuals are fine, and there are no tracking issues.

I'm using an Xbox One Elite controller with the wireless adapter. I've set the control options to use a standard controller, with standard stick movement. I was initially experiencing issues with the right stick not reliably turning, but was able to resolve this by setting **vr_deadzoneYaw 0** in the console.

Currently, the left stick does not produce movement. I can look down and see the model's legs moving, and can sometimes walk backwards, but the left stick doesn't reliably control the character. WASD does not have the same issue, and appears to work fine. Any help is appreciated, thanks in advance!

**Comment by KozGit** [comment]
Have you tried re-binding the left stick in the controls->key bindings menu?  

Are you using freepie in conjunction with Trinus?

**Comment by OpaqueEcho** [comment]
I was able to eventually get the left stick to work after rebinding, per your recommendation; took a few attempts for it to take (is there an easy way to clear all bindings for a given action? It seems that controller buttons are frequently bound to multiple functions by default).

Currently, it appears that changes to the horizontal and vertical controller sensitivity are not saved via the GUI; still testing.

 I'm also having issues where the tracking "flickers" periodically, for lack of a better term, but this is likely related to FreePIE HMD tracking; identical to what was reported here:
https://groups.google.com/forum/#!msg/psmoveservice/jJ3Z1HoCZ-I/UoshRh4IAwAJ

**Comment by KozGit** [comment]
Controller buttons should not be bound to more than one action by default - there may be more than one button assigned to an action, but each individual button should only be assigned to one action at a time.  If for some reason the config appears to be corrupt, just delete the config file from your save game directory, and the default for your headset should be recreated the next time you launch.

**Comment by OpaqueEcho** [comment]
Yeah, it was odd, (Y) was bound for interacting with objects, but was also bound to Next Weapon (I am 100% confident that I did not configure things this way). Thank you for your help, I'm really excited to dig into this once I get the tracking quirks sorted out!
